includes/TaskManagers/PluginInstallTaskManager.php

```diff
@@ -174,7 +174,7 @@ public static function add_to_queue( PluginInstallTask $plugin_install_task ) {
 		 return \update_option( Options::get_option_name( self::$queue_name ), $queue->to_array() );
 	}
 
-	public static function remove_from_queue( $plugin ) {
+	protected static function remove_from_queue( $plugin ) {
```

Comment:
The function is also used outside this class in `PluginUninstallTaskManager` that does not extend this class, pretty sure this is also a fatal error.